src/strategies/hashes-voting/index.ts

```diff
@@ -0,0 +1,55 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'Arr00';
+export const version = '0.1.0';
+
+const abi = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: '_account',
+        type: 'address'
+      }
+    ],
+    name: 'getCurrentVotes',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```

Comment:
Can you change to this:
```suggestion
const abi = ['function getCurrentVotes(address _account) view returns (uint256)'];
```